composer.json

```diff
@@ -72,6 +72,8 @@
         "sonata-project/doctrine-orm-admin-bundle": "^3.5",
         "sonata-project/notification-bundle": "^3.4",
         "sonata-project/seo-bundle": "^2.5",
+        "symfony/browser-kit": "^4.3",
+        "symfony/mailer": "^4.3",
```

Comment:
Seems like it should be swiftmailer: https://github.com/sonata-project/SonataNewsBundle/blob/e67fc7304c78de8469569e82499fbea6059198bf/src/Mailer/Mailer.php#L45

Reply:
It is currently added: `"swiftmailer/swiftmailer": "^4.3 || ^5.0 || ^6.0",`

Reply:
Then why is this needed?

Reply:
```
1) Sonata\NewsBundle\Tests\Functional\Controller\DashboardActionTest::testDashboard
Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException: The service "sonata.news.mailer" has a dependency on a non-existent service "mailer".
```

It is strange becouse even if `symfony/mailer` is require in some build is not installed:
https://travis-ci.org/github/sonata-project/SonataNewsBundle/builds/703645630

Reply:
You probably need swiftmailer-bundle instead?

Reply:
> ```
> 1) Sonata\NewsBundle\Tests\Functional\Controller\DashboardActionTest::testDashboard
> Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException: The service "sonata.news.mailer" has a dependency on a non-existent service "mailer".
> ```
> 
> It is strange becouse even if `symfony/mailer` is require in some build is not installed:
> https://travis-ci.org/github/sonata-project/SonataNewsBundle/builds/703645630

The `mailer` service was a swiftmailer in old symfony versions. We do not need the new `symfony/mailer`.

Reply:
swiftmailer-bundle will resolve this